doc/WebRTC.xml

```diff
@@ -634,6 +719,12 @@ Server -> Device: { "method": "trickle", "params": {"session": "s1", "candidate"
 Client -> Server: { "method": "trickle", "params": {"session": "s1", "candidate": {...}}}
 Server -> Device: { "method": "trickle", "params": {"session": "s1", "candidate": {...}}}
 ...
+Client -> Server: { "method": "extend", "params": {"session": "s1", "authorization": "...", "id": 4}

```

Comment:
Reviewing the rendering of this update on the developer portal this update extends off the page and thus would be truncated when rendered to a PDF. 

![Truncation](https://github.com/user-attachments/assets/2a040e8e-812b-40d3-9889-d35c79e31a59)